lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from = ::Time.at(0), to = ::Time.local(2050, 12, 31,  23,  23,  59, 999_999.999))
```

Comment:
I am still not sure which default value to use fot `to = `
Because ruby will accept anything, even Time.local(999999999999999, 12, 31,  23,  23,  59, 999_999.999).